src/mmw/mmw/settings/layer_settings.py

```diff
@@ -141,6 +141,7 @@
         'stream': True,
         'overlay': True,
         'minZoom': 5,
+        'maxZoom': 22,
```

Comment:
Is it possible to set `maxZoom` for this layer dynamically on the client side to === the `maxZoom` of whatever is the currently selected basemap?